README.md

```diff
@@ -91,19 +103,74 @@ See [example docker-compose.yml](docker-compose.yml).
 
 ## Run RCON commands
 
-Open a shell into your container via `docker exec -ti palworld-dedicated-server bash`, then you can run commands against the gameserver via the command `rcon` or `rconcli`
+Open a shell into your container via `docker exec -ti palworld-dedicated-server bash`, then you can run commands against the gameserver via the command `rconcli`
 
 ```shell
-$:~/steamcmd$ rcon showplayers
+$:~/steamcmd$ rconcli showplayers
```

Comment:
Unsure if we wanted here the rconcli or the rcon, need to talk about this again to refresh my memory.